.github/dependabot.yml

```diff
@@ -3,11 +3,11 @@ version: 2
 updates:
 
 - package-ecosystem: github-actions
-  directory: /
+  directory: /.github/workflows
```

Comment:
My thinking is that this will make dependabot skip the `requirements` directory. Don't know if this will work though...